### PR TITLE
docs: minor fixes to readme to help developers set up

### PR DESCRIPTION
**Pull request by kevin-kline**
This fixes README commands to start project. Also added grafana_icon.svg to correct directory.

package.lock.json is also committed as a first time commit - hence the large number of lines in this PR.

### CODE REVIEW

**Comment by klesh** [comment]
Hi, @kevin-kline , I saw some changes from `ts-main-doc` which had been merged, could you rebase `ts-main` ?

**Comment by joncodo** [comment]
@kevin-kline any idea why git always commits a new package.json for you?

File: plugins/jira/src/index.ts [hunk 1]
```diff
@@ -37,6 +38,9 @@ class Jira implements Plugin {
     console.info('Excute Jira', options);
     return;
   }
+  async collectData(primaryKeys: PrimaryValues, consumerModule) {
```
Comment (klesh) [inline plugins/jira/src/index.ts:41]: I think this method and PrimaryValues type should not be included in this PR.

**Review by klesh — COMMENTED** [review]
Hi, @kevin-kline , looking good overall, except some minor issue, I think we better keep this PR in `doc` level. 😊

**Comment by kevin-kline** [comment]
> @kevin-kline any idea why git always commits a new package.json for you?

if you mean package.lock.json I think the reason this time was that none existed at the time of committing, which I mentioned in the description. If you mean package.json, I'm not sure I understand.

File: plugins/jira/src/index.ts (hunk 1 above)
Comment (kevin-kline) [reply plugins/jira/src/index.ts]: I reverted the commit as per your feedback.

**Comment by klesh** [comment]
LGTM

File: .env.sample [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # Main Database Connetion String
-DB_URL=mysql://root:root@localhost:3306/lake_db
+DB_URL=postgresql://postgres:postgres@localhost:5432/lake
```
Comment (narrowizard) [inline .env.sample:2]: I think we will use mysql connection string in `.env.sample` instead of postgres. But the PR is merged. I'll fix it in another PR.

Comment (narrowizard) [reply .env.sample:2]: BTW, services declared in `devops` and documents should be updated as well. #237

Comment (klesh) [reply .env.sample:2]: I checked the `devops/docker-compose.yml` and it was still using postgresdb. There were some inconsistency, pls update docker-compose.yml as well, make sure whole process go through when you do that.👍
